### PR TITLE
Move Gameroom walkthrough scene 2

### PR DESCRIPTION
**Pull request by shannynalayna**

### CODE REVIEW

File: docs/0.4/examples/gameroom/walkthrough/images/create_gameroom_diagram.svg [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<!--
```
Comment (agunde406) [inline docs/0.4/examples/gameroom/walkthrough/images/create_gameroom_diagram.svg:1]: The copyright has changed. Pleas update to match.

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 2]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+   endpoint listed in the proposed circuit. If the node is already connected, the
+   peer connector returns "​Ok​". If the node is not connected, the peer connector
+   creates the connection and, if successful, adds the connection to the Splinter
+   network.
+
+2. After the connection has been created, a message exchange starts for peer
+   authorization (described in ​Appendix A​).
+
+3. When peer authorization succeeds, the Acme admin service is notified that the
+   Bubba Bakery node (​bubba-node-000​) has been successfully authorized. Acme and
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:716]: Font inconsistency: `bubba-node-000`

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 3]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+        used to sign a circuit proposal: the requester node ID (the "home node"
+        of the requester and location of that user's public key) and arbitrary
+        metadata (represented as key/value string pairs).
+
+      * The key permissions manager checks that a public key is authorized in a
+        specific role. In the case of "create circuit" requests, the signing
+        public key must be authorized for the "proposal" role.
+
+2. If the request is valid, the Acme admin service creates a `​CircuitProposal​`
+   and stores it in the ​AdminServiceShared.pending_changes​ field. The protobuf
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:788]: Font inconsistency: `AdminServiceShared.pending_changes​`

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 4]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+    requester: <public key of requester>
+    requester_node_id: acme-node-000
+    ```
+
+3. The Acme admin service creates a consensus proposal (a `​Proposal​` struct)
+  with the following contents:
+
+   * Proposal ID: the expected hash of the ​CircuitManagementPayload​ bytes
+
+   * Summary: the expected hash of the created ​CircuitProposal
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:839]: Font inconsistency: `CircuitProposal`

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 5]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+    ``` yaml
+    admin_message:
+      message_type: PROPOSED_CIRCUIT,
+      propose_circuit:
+        circuit_payload: <circuit_managment_payload>
+        expected_hash: <expected hash of CircuitProposal generated by payload>
+        required_verifiers: < bytes of the required verifiers from proposal>
+    ```
+
+    b. The ​AdminMessage ​is then wrapped in an ​AdminDirectMessage​, which enables
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:886]: Font inconsistencies: `AdminMessage` and `AdminDirectMessage`

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 6]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+##### I-2.5.3. Bubba Bakery node receives Circuit Create request from Acme node
+
+The Bubba Bakery Splinter node receives the network message from the Acme node
+and starts the process of "unwrapping" the message with a series of dispatchers.
+
+1. A dispatcher takes the message and passes it to the correct message handler
+   based on the message type of the message. Each dispatcher either handles the
+   message or forwards the message onto the next dispatcher.
+
+    a. The Bubba Bakery Splinter node passes the ​NetworkMessage​ to the network
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:931]: Font inconsistency: `NetworkMessage`

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 7]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+    `​ProposedCircuit` message and passes it to
+    `​AdminServiceShared.pending_circuit_payloads`.
+
+##### I-2.5.4. Bubba Bakery node validates CircuitManagementPayload
+
+The Bubba Bakery admin service validates the `​CircuitManagementPayload​` using the
+same steps as in ​section I-2.5.1​.
+
+1. The admin service verifies that the `​CircuitManagementPayload​` and the
+   included CircuitCreateRequest​ are valid. (For details, see ​section I-2.5.1,
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:963]: Font inconsistency: `CircuitCreateRequest`

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 8]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+  persistence TEXT NOT NULL,
+  routes TEXT NOT NULL,
+  durability TEXT NOT NULL,
+  circuit_management_type TEXT NOT NULL,
+  alias TEXT NOT NULL,
+  status TEXT NOT NULL,
+  created_time TIMESTAMP NOT NULL,
+  updated_time TIMESTAMP NOT NULL,
+);
+```
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:1095]: The original version used spaces to align the TEXT and PRIMARY KEY/NOT NULL "columns".  Is the spacing significant?

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 9]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+|`gameroom::acme-node-000::bubba-node-000::<UUIDv4>`|`Trust`|
+
+| `persistence` | `routes` | `durability` | `circuit_management_type` |
+| :--- | :--- | :--- | :--- |
+| `Any` | `Any` | `None` | `gameroom` |
+
+
+| `alias` | `status` | `created_time` | `updated_time` |
+| :--- | :--- | :--- | :--- |
+|`Acme + Bubba`|`pending`|`<time entry was created>`|`<time entry was updated>`|
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:1126]: These tables aren't rendering correctly. There's no format difference between the "header" rows and the content rows, and the space between tables isn't being shown. I wonder if HTML tags (\<table>, \<tr>, \<td>, etc.) would work better?

But tables are a pain - and might never look good on a small screen. 

Maybe this information should be presented in a different way, such as a code block with spaces to align information?

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 10]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+|`"peer_services": [ "gameroom_bubba-node-000" ], "admin_keys": ....`|`pending`|
+|`"peer_services": [ "gameroom_bubba-node-000" ], "admin_keys": ....`|`pending`|
+
+
+| `created_time` | `updated_time` |
+| :--- | :--- |
+|`<time entry was created>`|`<time entry was updated>`|
+|`<time entry was created>`|`<time entry was updated>`|
+
+##### I-2.8.3. New `gameroom_service` table entry
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:1236]: Wrong number and title: It should be I-2.3.4 (not a duplicate of the previous section).

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 11]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+| `proposal_type` | `circuit_hash` | `requester` | `requester_node_id` |
+| :--- | :--- | :--- | :--- |
+|`Create`|`<hash of circuit definition>`|`<public key of requester>`|`acme-node-000`|
+
+
+| `status` | `created_time` | `updated_time` |
+| :--- | :--- | :--- |
+| `pending` | `<time entry was created>` | `<time entry was updated>` |
+
+##### I-2.8.3. New `gameroom_notification` table entry
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:1285]: Wrong number and title: It should be I-2.3.5 (and mention gameroom_notification).

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 12]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+      }
+    ],
+    "roster": [{
+      "service_id": "gameroom_acme-node-000",
+      "service_type": "scabbard",
+      "allowed_nodes": [ "acme-node-000" ],
+      "arguments": {
+        "peer_services": [ "gameroom_bubba-node-000" ],
+        "admin_keys": [
+          <acme gameroomd public key>
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:1039]: Here, too, the maroon text on black is hard to read. If the text color can't be changed, perhaps we should lighten the backgrounds for all code snippets?

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 13]
```diff
@@ -388,3 +422,970 @@ GET /api/proposals
...
+to the Acme Splinter daemon. Before sending the proposal request, the Gameroom
+REST API asks the Gameroom UI to sign it with Alice's information.
+
+1. The Gameroom daemon uses the information from the "create new gameroom"
+   request to create a new ​CircuitManagementPayload​.
+
+    The following example shows a YAML representation of the
+    `CircuitManagementPayload​`.
+
+    ***Application metadata​***:
```
[the remark this answers is not in the document]
Comment (shannynalayna) [reply docs/0.4/examples/gameroom/walkthrough/index.md:549]: No, that is just a header to show what is being displayed. The line in the original document is not apart of that message, and renders badly here.

File: docs/0.4/examples/gameroom/walkthrough/index.md (hunk 8 above)
Comment (shannynalayna) [reply docs/0.4/examples/gameroom/walkthrough/index.md:1095]: Nope

File: docs/0.4/examples/gameroom/walkthrough/index.md (hunk 12 above)
Comment (shannynalayna) [reply docs/0.4/examples/gameroom/walkthrough/index.md:1044]: Updated all of these to remove the json specifier so the formatting looks nicer

File: docs/0.4/examples/gameroom/walkthrough/index.md (hunk 9 above)
Comment (shannynalayna) [reply docs/0.4/examples/gameroom/walkthrough/index.md]: Updated all of the tables to be generated with html

**Pull request merged**